spec/draft-barnes-cfrg-4q.md

```diff
@@ -73,25 +74,30 @@ informative:
       
 --- abstract
 
-This document specifies an elliptic curve over a quadratic extension
-of a prime field that offers the fastest known Diffie-Hellman key
-agreements while using compact keys. This high performance also extends
-to digital signature schemes. The best known attacks require 2^125 or so 
-operations, comparable to X25519, while performance is more than two
-times better.
+This document specifies an elliptic curve defined over a quadratic extension
+of a prime field that offers the fastest known Diffie-Hellman key agreements while using compact keys.  
+Similar to Curve25519, this curve is intended to target the 128-bit security level, but offering 
+performance that is more than two times better.  This curve is uniquely determined by a list of properties 
+that are defined by clear performance and security goals.
 
 --- middle
 
 # Introduction
 
 Public key cryptography continues to be computationally expensive particularly on less powerful devices. While
 recent advances have substantially reduced the cost of elliptic curve operations, the use of endomorphisms enables
-even more speedups, as does using quadratic fields over smaller primes.
+even more speedups, as does using quadratic fields over smaller primes.  This facilitates significant reductions in power 
+consumption as well as computing time.   
+
+This document specifies an elliptic curve ("Curve4Q"), proposed in [Curve4Q], that supports
+constant-time, exception-free scalar multiplications that are significantly faster and more power-efficient
+than any other available alternatives. As described in [Curve4Q], this elliptic curve is uniquely determined by specific 
+properties that are required to achieve top performance at a specific security level: Curve4Q is the only known curve that 
+permits a four dimensional decomposition over the highly efficient field GF(p^2) with p = 2^127 - 1 while offering close to 
+128 bits of security.  This "uniqueness" is an important feature, given the recent concerns about the provenance and generation 
```

Comment:
Are these concerns backed up by actual cryptanalysis? No. The concerns that are about the PITA that is writing constant time, secure implementations, and avoiding failure modes due to uncompressed points.